ocis-pkg/log/log.go

```diff
@@ -64,6 +70,19 @@ func NopLogger() Logger {
 	return Logger{zerolog.Nop()}
 }
 
+type LineInfoHook struct{}
+
+// Run is a hook to add line info to log messages.
+// I found the zerolog example for this here:
+// https://github.com/rs/zerolog/issues/22#issuecomment-1127295489
+func (h LineInfoHook) Run(e *zerolog.Event, l zerolog.Level, msg string) {
+	_, file, line, ok := runtime.Caller(3)
+	if ok {
+		file := pathRegex.ReplaceAllString(file, "")
```

Comment:
Do we really need this?  How does the log look for messages issued by packages not part for ocis. IIRC the `idp` service (coming from github.com/libregraph/lico) is using this logger (wrapped though logrus_wrapper.go) as well.

Reply:
Removed regex.